introduction.md

```diff
@@ -1 +1,5 @@
-Ho. Konnichiwa, sensei! My preference is for git, but html is up next.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?